### DISCUSSION TITLE
Adding missing dependencies to the installation guide (FreeBSD server)

### DISCUSSION THREAD

**Discussion by MatykoBr** [wavelog/wavelog#327] [Ideas]
I recently tried to install wavelog on a FreeBSD server, but it didnt work properly, althogh I followed every single step in the wiki. I also tried it on my PC (with XAMPP) and it worked without an issue. So I did a little bit of debugging and noticed, that I haven´t installed some important PHP-packages. After installing them, everything worked perfectly fine.

**The missing packages are:** _zlib, simplexml_

Normally theese packages are installed with PHP by default, but in my case, I had to install them manually. (I tested it with PHP 8.3 and 8.1)

Without theese packages installed, I can open the installation dialog, but when I click the Install button, I immediately get a HTTP 500 error. As mentioned above, after installing theese packeges, I could install wavelog properly.

So my feature request would be, that you update the wiki and add these three packages to the prerequisites adn update the command for installing PHP to something like this: `pkg install php81 php81-ctype php81-curl php81-mysqli php81-mbstring php81-session php81-xml php81-zip php81-zlib php81-simplexml php81-filter`

**Comment by HB9HIL** [comment]
Dear @MatykoBr 

Our Wiki can be edited by everyone 😄 
So if you want and you have some spare time we would really appreciate it if you can update the FreeBSD Wiki. 

https://github.com/wavelog/Wavelog/wiki/Installation-on-a-FreeBSD-Server

You would be our wave-hero!  🎉

**Reply by MatykoBr** [reply]
Done! 😉

**Reply by HB9HIL** [reply]
Thank you :)
